### PR TITLE
Small fixes and changes

### PR DESCRIPTION
**Pull request by TheRealKeto**
This PR changes the ignore file to use Github's, which generally ignores more files which are unnecessary, such as default virtual envs. Other changes have been documented in commit messages.

### CODE REVIEW

**Comment by m1stadev** [comment]
Denying this, as I actually prefer AutoTSS to mention you when replying to a message, as it helps the message stand out if you're doing something interactive (e.g. if you're adding a device), and other users are sending messages in the channel.

**Comment by TheRealKeto** [comment]
At least cherry-pick the commit with the gitignore, as well as other changes which clean up your code.

**Comment by TheRealKeto** [comment]
Decided to revert the whole reply mentions part; this PR still has some small but critical changes.

**Comment by m1stadev** [comment]
The PR looks good to me, but I'm gonna nitpick a few things.

**Comment by TheRealKeto** [comment]
Edits can be made directly in the PR; I enabled the option before making the PR.

File: bot.py [hunk 1]
```diff
@@ -42,10 +42,22 @@ def main():
 
     check_tsschecker()
 
-    intents = discord.Intents.default()
-    intents.members = True
-
-    client = commands.AutoShardedBot(command_prefix=get_prefix, help_command=None, intents=intents)
+    mentions = discord.AllowedMentions(
+        **dict.fromkeys([
```
Comment (m1stadev) [inline bot.py:46]: I'd prefer if `roles` and `everyone` were just passed as args into `AllowedMentions`, rather then using a dict, as it's a bit unnecessary.

**Review by m1stadev — CHANGES_REQUESTED** [review]

File: bot.py (hunk 1 above)
Comment (TheRealKeto) [reply bot.py:46]: While it uses more lines, it's generally more readable; it circumvents the need to scroll on your IDE to the right to see the entire line. This has already been removed though.

**Pull request merged**